fast64_internal/f3d/f3d_writer.py

```diff
@@ -169,12 +192,9 @@ def fixLargeUVs(obj):
         if material.f3d_mat.use_large_textures:
             continue
 
-        f3dMat = material.f3d_mat
-
-        UVinterval = [
-            2 if f3dMat.tex0.S.mirror or f3dMat.tex1.S.mirror else 1,
-            2 if f3dMat.tex0.T.mirror or f3dMat.tex1.T.mirror else 1,
-        ]
+        # To prevent wrong UVs when wrapping UVs into valid bounds,
+        # we need to account for the highest texture shift and if mirroring is active.
```

Comment:
To reiterate on my previous comments why "highest"? There must be a piece of code elsewhere on exporting that handles this similarly? Where